website_sale_cart_preview/static/src/css/website_sale_cart_preview.css

```diff
@@ -0,0 +1,26 @@
+.mycart-popover .cart_line {
```

Comment:
Because I never wrote .less and I never saw an OCA module using it. But with an example I can do it.